eva-remapping-get-source/src/test/java/uk/ac/ebi/eva/remapping/source/batch/processors/VariantToVariantContextProcessorTest.java

```diff
@@ -65,6 +66,7 @@ private SubmittedVariantEntity buildVariant(String chr1, int start, String refer
         SubmittedVariantEntity variant = new SubmittedVariantEntity(SS_ID, "hash", "assembly", taxonomy, project, chr1,
                                                                     start, reference, alternate, RS_ID,
                                                                     false, false, false, false, 1);
+        variant.setCreatedDate(LocalDateTime.now());
```

Comment:
Are there downsides to using a single timestamp per run for easier debugging?

Reply:
Done